spec/server.rb

```diff
@@ -15,8 +15,13 @@
   gc_after_request_time: 0.1,
   kill_after_requests: killer == 'requests' && 5,
   kill_on_too_much_memory: killer == 'memory' && {max: 0, check_every: 3},
+  map_term_to_quit: ENV["FINISH_REQUESTS"],
```

Comment:
`map_term_to_quit: ENV["MAP_TERM_TO_QUIT"]`
Would make things a bit more readable in this specs.